tailwind.config.js

```diff
@@ -0,0 +1,5 @@
+/** @type {import('tailwindcss').Config} */
+module.exports = {
+  presets: [require('./libs/ui/tailwind.config.js')],
+  darkMode: ['class', '.dark-mode'],
```

Comment:
I think `dark-mode` is default class, it's not necessary here
```suggestion
  darkMode: 'class',
```

Reply:
default class is `.dark`

<img width=300 src="https://github.com/kodadot/nft-gallery/assets/10708802/80bf1844-01a9-4ea4-bfe1-f3188eafc8de" >

https://tailwindcss.com/docs/dark-mode#toggling-dark-mode-manually

Reply:
alright I would change this in a follow up
- `light-mode` => `light`
- `dark-mode` => `dark`